packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -13,6 +13,7 @@ import 'terminal.dart';
 // ignore_for_file: non_constant_identifier_names
 
 const String fire = '🔥';
+const String image = '🖼️';
```

Comment:
Rewrite for Windows terminals that don't support emoji here:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/base/logger.dart#L366

Reply:
Done